src/gc/gc.cpp

```diff
@@ -5157,29 +5157,27 @@ void set_thread_affinity_mask_for_heap(int heap_number, GCThreadAffinity* affini
             {
                 if (bit_number == heap_number)
                 {
-                    dprintf (3, ("Using processor %d for heap %d\n", proc_number, heap_number));
+                    printf ("Using processor %d for heap %d\n", proc_number, heap_number);
```

Comment:
I guess this change from dprintf to printf was made just for your debugging purposes.